src/components/SectionDesignProfessionals.js

```diff
@@ -1,27 +1,35 @@
 import React from 'react';
-import styled from 'styled-components';
+import styled from 'styled-components/macro';
 import { H2, H6, TextSm } from './Typography';
 import Section from './Section';
 import image from '../images/designprofessionals.svg';
+import Container from './Container';
 
 const Wrapper = styled.div`
-  margin-bottom: 100px;
+  margin-bottom: 30px;
+
+  @media screen and (min-width: 450px) {
+    margin-bottom: 60px;
+  }
+
+  @media screen and (min-width: 850px) {
+    margin-bottom: 100px;
+  }
```

Comment:
As this is the same in every section, move this to the `Section` component

Reply:
Done